python/src/main/python/drivers/glsl-to-spv-worker.py

```diff
@@ -216,18 +222,21 @@ def do_image_job(args, image_job):
             return res
 
     remove(png)
-    remove(vkrun.LOGFILE)
+    remove(runspv.LOGFILE_NAME)
 
-    if args.linux:
-        vkrun.run_linux(vert_spv_file, frag_spv_file, json_file, skip_render)
-    elif args.vkrunner:
-        vkrun.run_vkrunner(vert_spv_file, frag_spv_file, json_file)
+    if args.legacy_worker:
+        if args.target == 'host':
+            runspv.run_image_android_legacy(vert=vert_spv_file, frag=frag_spv_file, json=json_file,
```

Comment:
This should be `run_image_host_legacy`?

Reply:
I fixed this but it would be good to double check

Reply:
Looks good